server/bin/gold/test-5.2.txt

```diff
@@ -356,28 +356,26 @@ Summary Statistics for Tar Balls on 1970-01-01 (external data only):
                     1.8 kB size
 
         By Server Origin:
-                         3 "ONE" pbench satellite server
-                         5 main pbench server
+                         8 main pbench server
 
         Controller Counts:
                          6 controllers
-                         3 "ONE" pbench satellite server
-                         3 main pbench server
+                         6 main pbench server
```

Comment:
Are we sad to lose these reporting distinctions?  Are we sad enough to try to do anything about it?

Reply:
I don't know after removing the prefix from the controller name how we will be able to distinguish between a dataset at audit level. I am not sure if we want to pursue this.

Reply:
This *will* be possible, after #3194 lands, and `pbench-results-move` and then put-shim leverage it, if we make the audit mechanism read the metadata. Maybe we just worry about it then. The whole "audit" process as current implemented is really archaic anyway, and needs to be completely rebuilt to consider SQL, Elasticsearch, Audit log, and so forth. I don't think this is worth worrying about now.